metadata.yaml

```diff
@@ -27,6 +33,14 @@ requires:
   prometheus-target:
     interface: http
   prometheus-rules:
-    interface: prometheus-rules
+    interface: prometheus_rules
   dashboards:
-    interface: grafana-dashboard
+    interface: grafana_dashboard
+  nrpe-external-master:
+    interface: nrpe_external_master
```

Comment:
The interface name in the NRPE charm uses dashes: https://git.launchpad.net/charm-nrpe/tree/metadata.yaml#n21